lib/puppet/application/lookup.rb

```diff
@@ -57,7 +57,6 @@ class Puppet::Application::Lookup < Puppet::Application
     options[:node] = arg
   end
 
-  # not yet supported
   option('--facts FACT_FILE') do |arg|
     if %w{.yaml .yml .json}.include?(arg.match(/\.[^.]*$/)[0])
```

Comment:
Perhaps prudent to warn the user that the argument is ignored (and why) when it doesn't match the given regexp.